synapse/rest/admin/v1/__init__.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
```

Comment:
please don't do this; let's sort the modules by functionality, not version.

Reply:
(also lumping a massive change like this into this PR makes it impossible to see what is actually changing)

Reply:
So this will change also for the client endpoints in the future?

Reply:
maybe one day. But let's not copy the mistakes of `/rest/client` into `/rest/admin`.